instagram-page-react/src/pages/profilePage/actions/profilePage.posts.js

```diff
@@ -0,0 +1,21 @@
+// services
+import { fetchProfilePagePosts } from "../services/profilePage.posts";
+
+// actions
+import { saveProfilePagePostsAction } from "./actionCreators/profilePage.posts";
+
+const handleSuccessFetchPosts = (dispatch, result) => {
+    dispatch(saveProfilePagePostsAction(result));
+};
+
+const handleErrorFetchPosts = (err) => {
+    alert(err.message);
+};
+
+export const fetchPosts = () => {
+    return (dispatch) => {
+        return fetchProfilePagePosts()
+        .then((result) => handleSuccessFetchPosts(dispatch, result))
+        .catch((err) => handleErrorFetchPosts(err));
```

Comment:
We don't need to create an extra function here, we can directly pass:
`.catch(handleFetchPostsError)`

Also, `handleSuccessFetchPosts` <> `handleFetchPostsSuccess` and `handleErrorFetchPosts` <> `handleFetchPostsError`